wallets/rn_cli_wallet/src/components/GetStartedButton.tsx

```diff
@@ -18,14 +32,56 @@ export function GetStartedButton() {
     });
   };
 
+  async function registerAccount() {
+    if (!notifyClient || !account) {
+      console.warn('notify client or account not available');
+      return;
+    }
+
+    try {
+      const {message, registerParams} = await notifyClient.prepareRegistration({
+        account,
+        domain: '',
+        allApps: true,
+      });
+
+      const {eip155Wallets} = await createOrRestoreEIP155Wallet();
```

Comment:
Also, what do you think of moving all the logic to the view instead of having it inside a button? So we can keep the components 'dumb' and not so tied to a specific logic

Reply:
Good call. I found here that `useInitialization` is not a quite descriptive name. Since it's initializing the w3w, what about changing it to `useInitializeWeb3Wallet`?

> what do you think of moving all the logic to the view
Totally, event more the component shouldn't have any business logic. We could move all business logics into utils and hooks, if they have state that needs to be drilled to children, a global state could be used. I'll refactor the logic a bit here.

Reply:
@ignaciosantise Refactored the hooks we have two initialization hooks now:
- useIntializeWeb3Wallet
- useInitializeNotifyClient

Now the Notify Clients that created wallet and initialize once it's ready and initializes the Notify Client. For this, I needed to store the created wallet into somewhere and used `SettingsStore` since we're [storing wallet address there](https://github.com/WalletConnect/react-native-examples/pull/124/files#diff-f150179cf1bd9bb4f832013ad4defe4a45217a3754c111307e212aceaff0f1e1R20), I think it's not a perfect place to store but let's keep it unless we don't need similar things in global state soon.